ads/google/adsense.js

```diff
@@ -17,6 +17,21 @@
 import {validateData} from '../../3p/3p';
 import {setStyles} from '../../src/style';
 import {camelCaseToDash} from '../../src/string';
+import {user} from '../../src/log';
+
+/** @const {Array<number>} */
+const WHITELISTED_HEIGHTS = [150];
+
+/**
+ * Check that the <amp-ad> tag height has not been tampered with.
```

Comment:
Nit: s/"has not been tampered with"/"is one of the allowed values for responsive ad units" (or similar)